eva-server/src/main/resources/application.properties

```diff
@@ -20,9 +20,9 @@ springfox.documentation.swagger.v2.path=/webservices/api
 #security.oauth2.resource.user-info-uri = ...
 
 spring.data.mongodb.host=@eva.mongo.host@
-spring.data.mongodb.authentication-database=@eva.mongo.auth.db@
-spring.data.mongodb.username=@eva.mongo.user@
-spring.data.mongodb.password=@eva.mongo.passwd@
+#spring.data.mongodb.authentication-database=@eva.mongo.auth.db@
+#spring.data.mongodb.username=@eva.mongo.user@
+#spring.data.mongodb.password=@eva.mongo.passwd@
```

Comment:
it's ok for you to have this changes to test locally, but this can't be merged in the main branch because it won't work in production

Reply:
Okay, I will change it back before the merge :)